behat/symfony2-extension/2.1/Makefile

```diff
@@ -0,0 +1,5 @@
+behat-init:
+	@test -f bin/behat && bin/behat --init || echo "cannot init behat"
+
+behat:
+	@test -f bin/behat && bin/behat || echo "cannot run behat"
```

Comment:
What's the benefit of defining these in the Makefile instead of just using `bin/behat` directly?